test/core/transport/chttp2/graceful_shutdown_test.cc

```diff
@@ -431,7 +432,7 @@ TEST_F(GracefulShutdownTest, RequestStartedAfterFinalGoawayIsIgnored) {
   op++;
   error = grpc_call_start_batch(s, ops, static_cast<size_t>(op - ops), Tag(101),
                                 nullptr);
-  GPR_ASSERT(GRPC_CALL_OK == error);
+  CHECK_EQ(error, GRPC_CALL_OK);
```

Comment:
Just for my learning, why do we prefer CHECK_EQ(error, GRPC_CALL_OK) instead of CHECK(error == GRPC_CALL_OK) here? Thanks.

Reply:
Got it. Thanks for explaining.